packages/components/src/notice/style.scss

```diff
@@ -1,27 +1,33 @@
-.components-notice {
-	background-color: $blue-medium-100;
-	border-left: 4px solid $blue-medium-500;
-	margin: 5px 15px 2px;
-	padding: 8px 12px;
-
-	&.is-dismissible {
-		padding-right: 36px;
-		position: relative;
-	}
+.edit-post-layout {
```

Comment:
Why is this className needed here? Ideally if the fix here is generic, we can do it without this className, if it's specific to the notices used in the editor, we should override the default styles in the `edit-post` package instead.

Reply:
That className is needed because the original class of `.components-notice.is-dismissible`, which adds the right padding, gets overridden by this `.edit-post-layout .components-notice-list .components-notice`. So the idea was to more specifically target the dismissable notices with `.edit-post-layout .components-notice-list .components-notice.is-dismissible`. You can see the issue here #11156. I'm all for putting this in another place if that makes more sense, I just wasn't sure where it should go.